packages/metals-vscode/src/extension.ts

```diff
@@ -589,22 +589,32 @@ function launchMetals(
       registerCommand(
         ClientCommands.StartDebugSession,
         (param: ScalaCodeLensesParams) => {
-          scalaDebugger.start(false, param).then((wasStarted) => {
-            if (!wasStarted) {
-              window.showErrorMessage("Debug session not started");
+          scalaDebugger.start(false, param).then(
+            (wasStarted) => {
+              if (!wasStarted) {
+                window.showErrorMessage("Debug session not started");
+              }
+            },
+            (reason: Error) => {
+              window.showErrorMessage(reason.message);
```

Comment:
did you check that typeof reason in runtime is `Error`? Where does it come from?

Reply:
That's what I was getting in runtime, should I do a check to make sure it's an Error?

Reply:
I added a check just in case.